site/src/github/comparison_summary.rs

```diff
@@ -221,65 +227,92 @@ async fn summarize_run(
         }
     }
 
+    let bootstrap = summarize_bootstrap(&inst_comparison);
+
     let metrics = vec![
         (
             "Instruction count",
             Metric::InstructionsUser,
-            true, // highly reliable
+            MetricReliability::High,
             inst_comparison,
         ),
         (
             "Max RSS (memory usage)",
             Metric::MaxRSS,
-            false, // not highly reliable
+            MetricReliability::Low,
             calculate_metric_comparison(ctxt, &commit, Metric::MaxRSS).await?,
         ),
         (
             "Cycles",
             Metric::CyclesUser,
-            false, // not highly reliable
+            MetricReliability::Low,
             calculate_metric_comparison(ctxt, &commit, Metric::CyclesUser).await?,
         ),
+        (
+            "Binary size",
+            Metric::LinkedArtifactSize,
+            MetricReliability::Low,
```

Comment:
Hmm, binary size if very reliable, but it shouldn't be shown by default in the comment. Maybe the enum should just be ShownByDefault/HiddenByDefault.

Reply:
:+1: